### PR TITLE
target 2.164.1

### PR DESCRIPTION
**Pull request by casz**
Here is our checklist for contributors. No hard requirement here, just a reminder

- [x] Please describe what you did

Let's discuss targeting the v2.164.1 LTS, the project is pretty stable.
There are a few PRs we could get in and release before bumping.
The bump would be welcomed this is, after all, JCasC we should be able to safely bump core version without too much fuss from Jenkins users.
Would be great to know we can run on Java 11 runtime!

- [ ] Link to issue you're working on if there's a relevant one

- [x] Did you provide a test-case to demonstrate feature actually works / issue is fixed ?

- [ ] Please also consider adding a line to [CHANGELOG.md](https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/CHANGELOG.md)

Changelog will be added later.

### CODE REVIEW

File: plugin/src/main/java/io/jenkins/plugins/casc/core/AdminWhitelistRuleConfigurator.java [hunk 1]
```diff
@@ -56,8 +56,13 @@ public String getName() {
 
     @Override
     protected AdminWhitelistRule instance(Mapping mapping, ConfigurationContext context) {
-        Injector injector = Jenkins.getInstance().getInjector();
-        return injector.getInstance(AdminWhitelistRule.class);
+        Injector injector = Jenkins.get().getInjector();
```
Comment (casz) [inline plugin/src/main/java/io/jenkins/plugins/casc/core/AdminWhitelistRuleConfigurator.java]: I wonder why injector is marked `@CheckForNull`, is there a better way of getting the AdminWhitelistRule instance?

Comment (timja) [reply plugin/src/main/java/io/jenkins/plugins/casc/core/AdminWhitelistRuleConfigurator.java]: also wondered the same thing in my PR when i had to do a similar change

File: pom.xml [hunk 2]
```diff
@@ -22,7 +22,7 @@
   <properties>
     <revision>1.9</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.60.3</jenkins.version>
+    <jenkins.version>2.164.1</jenkins.version>
```
Comment (timja) [inline pom.xml]: integrations/pom.xml should have the jenkins.version removed

Comment (casz) [reply pom.xml]: Plenty of deprecated functions to go through as well :smile:

**Comment by casz** [comment]
Whoops, I was actually meant to put this on JenkinsCi repo :smile:

**Pull request closed without merging**